docs: Add new section "Running a Node" (#7701)
cosmos/cosmos-sdk: docs/run-node/interact-node.md

<!--
order: 3
-->

# Interacting with the Node

## Pre-requisite Readings

- [Running a Node](./run-node.md) {prereq}

## Via CLI

Now that your chain is running, it is time to try sending tokens from the first account you created to a second account. In a new terminal window, start by running the following query command:

```bash

> simd query account $(simd keys show my_validator -a) --chain-id my-test-chain

Reviewer: would be good to use variables instead repeating the sub-commands:
```
export MY_VALIDATOR=$(....)

Reviewer 2: Yeah I agree. I'd call it `MY_VALIDATOR_ADDRESS` for the sake of clarity. The gist of it is that shorter examples are better than longer ones.

```

You should see the current balance of the account you created, equal to the original balance of `stake` you granted it minus the amount you delegated via the `gentx`. Now, create a second account:

```bash

> simd keys add receiver

Reviewer: ```suggestion
simd keys add receiver
export RECEIVER=$(simd keys show receiver -a)
```

Reviewer 2: We don't need to use the `export` statement though. For the sake of clarity and if we assume that the user runs all the commands in one terminal window/session, we can remove `export`.

```

> The command above creates a local key-pair that is not yet registered on the chain. An account is registered the first time it receives tokens from another account. Now, run the following command to send tokens to the second account:

Reviewer: ```suggestion
The command above creates a local key-pair that is not yet registered on the chain. An account is created the first time it receives tokens from another account. Now, run the following command to send tokens to the `receiver` account:
```

Reviewer 2: I'd call it the `recipient`. Not a big deal either way

Reviewer 2: FTR `recipient` belongs to a finance-friendly terminology, so...

```bash

> simd tx send $(simd keys show my_validator -a) $(simd keys show receiver -a) 1000stake --chain-id my-test-chain

Reviewer: ```suggestion
simd tx send $MY_VALIDATOR  $RECEIVER 1000stake --chain-id my-test-chain
```

```

Check that the second account did receive the tokens:

```bash
simd query account $(simd keys show receiver -a) --chain-id my-test-chain

> ```

Reviewer: we could paste this command in the code block above, would make the whole think shorter.
